### PR TITLE
refine: misc refines

### PR DESCRIPTION
**Pull request by baurine**
## What Did

- Handle get alert error custom, not show notification error
- Show statement/slow query search result count
- Show empty status for slow query

## Preview

image

image

image

### CODE REVIEW

File: ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/pages/List/index.tsx [hunk 1]
```diff
@@ -9,13 +9,16 @@ import {
   Menu,
   Dropdown,
   Alert,
-  Tooltip
+  Tooltip,
+  Result,
+  Button
```
Comment (YiniXu9506) [inline ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/pages/List/index.tsx:14]: Seems the `Button` not been used.

**Review by YiniXu9506 — APPROVED** [review]
LGTM

File: ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/en.yaml [hunk 2]
```diff
@@ -109,6 +109,10 @@ slow_query:
     status:
       success: Success
       error: Failed
+  overview:
+    empty_result: Great, the cluster has no any related slow queries!
```
Comment (shhdgit) [inline ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/en.yaml:113]: Why do we congratulate the user when they did not get the result they want? How about showing empty result tips only?

**Review by shhdgit — APPROVED** [review]
The rest LGTM.

File: ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/en.yaml (hunk 2 above)
Comment (baurine) [reply ui/packages/tidb-dashboard-lib/src/apps/SlowQuery/translations/en.yaml:113]: Is it not a good news if the cluster has no slow queries? ^_^

**Pull request merged**